interface/wx/grid.h

```diff
@@ -4188,21 +4268,29 @@ class wxGrid : public wxScrolledWindow
         you use this function in a mouse event handler you need to translate
         the mouse position, which is expressed in device coordinates, to
         logical ones.
+        The coordinates have to be relative to @a gridWindow.
```

Comment:
If the `gridWindow` parameter is valid, them the coordinates have to be inside that window.

For example if we have a frozenGridWindow of (width_x, 20), and a cell size of (30, 10):
`XYToCell(5, 5, frozenGridWindow)` returns cell 0,0
`XYToCell(5, 5, gridWin)` returns cell 0,2
`XYToCell(5, 5, NULL)` returns cell 0,0

`NULL` is the special case when we execute the function as if no cell is frozen, even if there are some. It provide the functionality from before introducing the gridWindow arg.

Reply:
Do we still use the functionality with `NULL` argument in `wxGrid` code? If not, then I think it would be better to not provide it at all. If the worry is about compatibility, then it's enough to just have `XYToCell(x, y)` forwarding to `XYToCell(x, y, m_gridWindow)`.